src/index.js

```diff
@@ -63,7 +63,7 @@ function checkReturnForState(editorState, ev, { insertEmptyBlockOnReturnWithModi
       newEditorState = changeCurrentBlockType(newEditorState, type, text.replace(/\n```\s*$/, ''));
       newEditorState = insertEmptyBlock(newEditorState);
```

Comment:
I'm not sure why this `if` behavior even exists (I don't think you wrote it), but I figured I would keep it.

As far as I can tell it says: "if your code block is just ` ```  ` and you hit enter, it should break out".